Firestore/Protos/build_protos.py

```diff
@@ -121,8 +121,14 @@ def run(self):
 
     self.__run_generator(nanopb_out)
 
-    sources = collect_files(nanopb_out, '.nanopb.h', '.nanopb.c')
-    post_process_files(sources, add_copyright, nanopb_rename_delete)
+    sources = collect_files(nanopb_out, '.nanopb.h', '.nanopb.cc')
+    post_process_files(
```

Comment:
I don't understand--why is this here? This was merged into master with #2046.

https://github.com/firebase/firebase-ios-sdk/blob/master/Firestore/Protos/build_protos.py#L125

Reply:
I think its a GitHub issue with cherry-pick merging. As far as I can tell the diff is not there.  Maybe github just wants us to know the change is coming on this branch too?